CHANGELOG.md

```diff
@@ -17,11 +17,20 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased](https://github.com/o1-labs/snarkyjs/compare/a632313a...HEAD)
 
+### Breaking changes
+
+- Remove optional `zkappKey` argument in `smartContract.init()`, and instead assert that `provedState` is false when `init()` is called https://github.com/o1-labs/snarkyjs/pull/908
+  - Breaks deployed zkApps which make `init()` a `@method`
+
 ### Changes
 
 - New decorator `CatchAndPrettifyStacktrace` for printing better stack traces thrown from SnarkyJS https://github.com/o1-labs/snarkyjs/pull/890
   - Cleans up stack traces by filtering out extra details, making them more readable
 
+### Fixes
+
+- Fix running SnarkyJS in Node.js on Windows https://github.com/o1-labs/snarkyjs-bindings/pull/19 (@wizicer)[https://github.com/wizicer]
+
```

Comment:
Unrelated: this points to latest `snarkyjs-bindings` with a fix for windows https://github.com/o1-labs/snarkyjs-bindings/pull/19